src/main/java/io/reactivex/plugins/RxJavaPlugins.java

```diff
@@ -186,10 +187,10 @@ public static boolean isLockdown() {
      * @param defaultScheduler the hook's input value
      * @return the value returned by the hook
      */
-    public static Scheduler initComputationScheduler(Scheduler defaultScheduler) {
+    public static Scheduler initComputationScheduler(Callable<Scheduler> defaultScheduler) {
```

Comment:
This was done for consistency with the expectations in `RxJavaPluginsTest.clearIsPassthrough()`, specifically:
```java
assertNull(RxJavaPlugins.initComputationScheduler(null));

            assertNull(RxJavaPlugins.initIoScheduler(null));

            assertNull(RxJavaPlugins.initNewThreadScheduler(null));

            assertNull(RxJavaPlugins.initSingleScheduler(null));
```

Should this be changed to only return null if the `Callable` returns null (and throw if the `Callable` itself is null)?